deploy/olm-catalog/cloud-resources/0.18.0/cloud-resources.clusterserviceversion.yaml

```diff
@@ -0,0 +1,291 @@
+          "metadata": {
+            "labels": {
+              "productName": "ProductName"
+            },
+            "name": "example-blobstorage"
+          },
+          "spec": {
+            "secretRef": {
+              "name": "example-blobstorage-sec"
+            },
+            "tier": "development",
+            "type": "REPLACE_ME"
+          }
+        },
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "Postgres",
+          "metadata": {
+            "labels": {
+              "productName": "productName"
+            },
+            "name": "example-postgres"
+          },
+          "spec": {
+            "secretRef": {
+              "name": "example-postgres-sec"
+            },
+            "tier": "development",
+            "type": "REPLACE_ME"
+          }
+        },
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "PostgresSnapshot",
+          "metadata": {
+            "name": "example-postgressnapshot"
+          },
+          "spec": {
+            "resourceName": "REPLACE_ME"
+          }
+        },
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "Redis",
+          "metadata": {
+            "labels": {
+              "productName": "productName"
+            },
+            "name": "example-redis"
+          },
+          "spec": {
+            "secretRef": {
+              "name": "example-redis-sec"
+            },
+            "tier": "development",
+            "type": "REPLACE_ME"
+          }
+        },
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "RedisSnapshot",
+          "metadata": {
+            "name": "example-redissnapshot"
+          },
+          "spec": {
+            "resourceName": "REPLACE_ME"
+          }
+        }
+      ]
+    capabilities: Basic Install
+    operators.operatorframework.io/builder: operator-sdk-unknown
+    operators.operatorframework.io/project_layout: go
+  name: cloud-resources.v0.18.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: BlobStorage is the Schema for the blobstorages API
+      kind: BlobStorage
+      name: blobstorages.integreatly.org
+      version: v1alpha1
+    - description: Postgres is the Schema for the postgres API
+      kind: Postgres
+      name: postgres.integreatly.org
+      version: v1alpha1
+    - description: PostgresSnapshot is the Schema for the postgressnapshots API
+      kind: PostgresSnapshot
+      name: postgressnapshots.integreatly.org
+      version: v1alpha1
+    - description: Redis is the Schema for the redis API
+      kind: Redis
+      name: redis.integreatly.org
+      version: v1alpha1
+    - description: RedisSnapshot is the Schema for the redissnapshots API
+      kind: RedisSnapshot
+      name: redissnapshots.integreatly.org
+      version: v1alpha1
+  description: Cloud Resources description. TODO.
```

Comment:
```suggestion
  description: Provision and manage in-cluster and cloud provider resources (Blob Storage, Postgres, Redis)
```

Reply:
@aidenkeating I'm just wondering, before the operator-sdk update. Was this info also manually populated or was it automatically generated?

Reply:
@darahayes I was just looking at this myself. In the previous Operator SDK we used, we could provide a `from version` flag that would derive the new CSV from a previous version. I think we've basically being pulling descriptions from version to version via that approach.

It doesn't look like we can leverage that approach anymore. Hopefully there's now an alternative